debian/debian_swraid_lvm_luks.md

```diff
@@ -16,41 +16,45 @@ This guide *could* work at any other provider with a rescue system.
 
 - Boot to the rescue system via hetzners server management page
 - install a minimal Debian 9.4 with hetzners "installimage" skript (https://wiki.hetzner.de/index.php/Installimage)
-- I choosed the following logical volumes on my system to keep it simple:
+- I chose the default raid1/lvm setup to keep it simple:
 
 ```
-lv-home (60GB) ext4
-lv-log (30GB) ext4
-lv-swap (10GB) swap
-lv-root (all) -> means remaining space ext4
+cat configs/simple-debian64-raid-lvm
```

Comment:
Is this hetzner specific? If yes, we should probably keep the old configuration block - however, a hint to the default setup seems reasonable to me

Reply:
Yes it is!